src/content/docs/cheerpj3/11-guides/CheerpJ-explanation.mdx

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
I don't like how 'CheerpJ' is in the file path / url of this page, but "explanation" on its own is too vague. Also `guides` is not the right folder for this.

Maybe `14-explanation/architecture.mdx`?